packages/database/package.json

```diff
@@ -51,5 +51,5 @@
   "bugs": {
     "url": "https://github.com/firebase/firebase-js-sdk/issues"
   },
-  "typings": "dist/index.d.ts"
+  "typings": "dist/esm/index.d.ts"
```

Comment:
Is this actually correct / useful?  All that file contains is:

```
import { FirebaseNamespace } from '@firebase/app';
export declare function registerDatabase(instance: FirebaseNamespace): void;
```

which is _not_ the typings for Firebase Realtime Database, right?

Reply:
This is not the curated typings no. This is the generated typings which is what we eventually want to move to. Just the first step on the way there.

Reply:
But good call on the firestore typings. I'll get that added.